server/chat-plugins/datasearch.ts

```diff
@@ -146,7 +146,7 @@ export const commands: ChatCommands = {
 		return runSearch({
 			tar: targetsBuffer.join(","),
 			cmd: 'randmove',
-			canAll: (!this.broadcastMessage || (room && room.isPersonal)),
+			canAll: (!this.broadcastMessage || (room?.isPersonal)),
```

Comment:
Probably take out the extra brackets here

Reply:
```suggestion
			canAll: (!this.broadcastMessage || room?.isPersonal),
```